### ISSUE TITLE
unlocking items

### ISSUE THREAD

**Issue by jpkj123** [prodzpod/RealerCheatUnlocks#3]
I unlocked all the items in the logbook but i still cant select them in game. please fix

**Comment by prodzpod** [comment]
can you provide log? cannot reproduce

**Comment by jpkj123** [comment]
![Risk of Rain 2 Screenshot 2024 09 17 - 23 13 16 87](https://github.com/user-attachments/assets/43d91a16-061e-410b-8dcb-812726f7b5d6)
![Risk of Rain 2 Screenshot 2024 09 17 - 23 14 33 51](https://github.com/user-attachments/assets/59abf9eb-0906-4a4b-850e-3504d0897d1e)
![Risk of Rain 2 Screenshot 2024 09 17 - 23 17 55 99](https://github.com/user-attachments/assets/1f521791-6b2d-41b8-82cd-d6c316b8f4bd)

**Comment by jpkj123** [comment]
also unlocked the items on page two just forgot to screenshot

**Comment by prodzpod** [comment]
yeah cannot repro still

**Comment by jpkj123** [comment]
I am using the artifact of command which allows me to choice my items in the menus like you see in the screenshot but even after unlocking all the items they where still locked in said menu

**Comment by prodzpod** [comment]
yeah i know youre using command, i unlocked the items from logbook and they appear on command menu for me

**Comment by jpkj123** [comment]
any idea what i should do?

**Comment by prodzpod** [comment]
try and see if there's any other mods causing it to break

**Comment by jpkj123** [comment]
i started a new profile with only this mod installed and its still not unlocking the items

**Comment by prodzpod** [comment]
yeah no it works on my end again, are you on a pirated copy? any dlcs you have?

**Comment by jpkj123** [comment]
im on steam and i dont have any dlcs

**Comment by Nias26** [comment]
Same problem here. Unlocks in the logbook and gives the achievemnt but doesn't show up
